lms/envs/production.py

```diff
@@ -1154,3 +1156,8 @@
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+
+######################### Secret Key for Google reCaptcha #####################
+
+CAPTCHA_SECRET_KEY = ENV_TOKENS.get('CAPTCHA_SECRET_KEY', None)
```

Comment:
Let's move this under the already existing `Philu Specific Settings` section, putting it at the bottom of that section with a simple comment as is already there for multiple other applications.

Reply:
Also, let's please not forget to prompt the DevOps that we need to add these tokens in production settings when regflow is deployed

Reply:
Also this should be moved to `auth.json` and used using `AUTH_TOKENS`